src/xdocs/config_blocks.xml

```diff
@@ -1045,6 +1045,24 @@ allowedFuture.addCallback(result -> {
 &lt;module name=&quot;RightCurly&quot;/&gt;
```

Comment:
This is a complex check since we check alot of different scenarios and some things are allowed while others are not.

Right now you only have basically 1 example per config. Can you expand examples more. What about ifs with no else, try/catch, methods, classes, etc...

Reply:
Sure np, working on this later this week